File: sh1_1.cpp
#include<iostream>
int sumSquares(int a,int b){
  int c=0;
  while(a<=b){
    c+=a*a;
    a++;
  }
  return c;
}
int main(){
  std::cout <<sumSquares(1,5);
  return 0;
}

File: sh1-1.java
public class sh1-1 {
  public static void main(String[] args) {
    System.out.println(sumSquares(1, 5));
  }
  static int sumSquares(int a,int b){
    int total=0;
    for (;a<=b;a++){
      total+=Math.pow(a, 2);
    }
    return total;
  }
}
